src/index.js

```diff
@@ -2,19 +2,51 @@ import express from 'express'
 import { ApolloServer } from 'apollo-server-express'
 import { createServer } from 'http'
 
+import passport from 'passport'
+import { Strategy as JwtStrategy, ExtractJwt } from 'passport-jwt'
+import jwt from 'jsonwebtoken'
+
 import schema from './schema'
+import models from './models'
 
 require('dotenv').config()
 
 const port = process.env.PORT || 3001
 
+const params = {
+  secretOrKey: 'secret',
+  jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken()
+}
+
+const strategy = new JwtStrategy(params, async (payload, done) => {
+  const user = await models.user.findOne({ where: { id: payload.sub } })
+  return done(null, user)
+})
```

Comment:
Use `try/catch` for the `findOne` call and report the error through the `done` callback:
```suggestion
const strategy = new JwtStrategy(params, async (payload, done) => {
  try {
    const user = await models.user.findOne({ where: { id: payload.sub } })
    done(null, user)
  } catch (err) {
    done(err)
  }
})
```